hedera-node/src/main/java/com/hedera/services/store/AccountStore.java

```diff
@@ -107,6 +127,36 @@ public Account loadAccountOrFailWith(Id id, @Nullable ResponseCodeEnum code) {
 		return account;
 	}
 
+	/**
+	 * Use carefully. Returns a model of the requested account which may be marked as detached.
```

Comment:
Given the fact, that this method was introduced to solve a very specific corner case - I think that we would benefit from more extensive JavaDoc, which provides more details on the specific context, covered by this implementation.